helm-charts/sep-service/templates/configmap.yaml

```diff
@@ -0,0 +1,135 @@
+          enabled: true
+        stellar:
+          enabled: false
+          name: "stellar"
+          horizonUrl: https://horizon-testnet.stellar.org 
+          secretKey: secret # stellar account secret key
+      circle-payment-observer:
+        enabled: true
+        horizonUrl: https://horizon-testnet.stellar.org
+        stellarNetwork: TESTNET # TESTNET or PUBLIC
+        trackedWallet: all
+      event:
+        # If enabled, publish Events to a queue (publisherType)
+        # publisherType - the type of queue to use for event publishing
+        enabled: true
+        publisherType: kafka
+      kafka.publisher:
+        bootstrapServer: {{ .Values.stellar.app_config.kafka_publisher.bootstrapServer }}
+        useSingleQueue: {{ .Values.stellar.app_config.kafka_publisher.useSingleQueue }}
+        eventTypeToQueue:
+          all: {{ .Values.stellar.app_config.kafka_publisher.eventTypeToQueue.all }}
+          quoteCreated: {{ .Values.stellar.app_config.kafka_publisher.eventTypeToQueue.quoteCreated }}
+          transactionCreated: {{ .Values.stellar.app_config.kafka_publisher.eventTypeToQueue.transactionCreated }}
+          transactionStatusChanged: {{ .Values.stellar.app_config.kafka_publisher.eventTypeToQueue.transactionStatusChanged }}
+          transactionError: {{ .Values.stellar.app_config.kafka_publisher.eventTypeToQueue.transactionError }}
+    data-spring-jdbc-sqlite:
+      spring.jpa.database-platform: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_jpa_database_platform }}
+      spring.jpa.hibernate.ddl-auto: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_jpa_hibernate_ddl_auto }}
+      spring.jpa.generate-ddl: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_jpa_generate_ddl }}
+      spring.jpa.hibernate.show_sql: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_jpa_hibernate_show_sql }}
+      spring.datasource.url: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_datasource_url }}
+      spring.datasource.driver-class-name: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_datasource_driver_class_name }}
+      spring.datasource.username: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_datasource_username }}
+      spring.datasource.password: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_datasource_password }}
+      spring.mvc.converters.preferred-json-mapper: {{ .Values.stellar.app_config.data_spring_jdbc_sqlite.spring_mvc_converters_preferred_json_mapper }}
+    data-spring-jdbc-h2:
+      spring.datasource.url: {{ .Values.stellar.app_config.data_spring_jdbc_h2.spring_datasource_url }}
+      spring.jpa.properties.hibernate.dialect: {{ .Values.stellar.app_config.data_spring_jdbc_h2.spring_jpa_properties_hibernate_dialect }}
+    spring:
+      logging:
+        level:
+          root: DEBUG
+          org.springframework: DEBUG
+          org.springframework.web.filter: DEBUG
+          org.stellar: DEBUG
```

Comment:
I'm not sure how verbose it is but maybe this should also be configurable via the values file?